EdgeMultiplay/Scripts/EdgeManager.cs

```diff
@@ -219,7 +235,7 @@ public async Task ConnectToServer(bool useAnyCarrierNetwork = true, bool useFall
                     integration.useFallbackLocation = useFallBackLocation;
                     wsClient = new MobiledgeXWebSocketClient();
                     await integration.RegisterAndFindCloudlet();
-                    integration.GetAppPort(LProto.L_PROTO_TCP);
+                    integration.GetAppPort(LProto.Tcp);
```

Comment:
is the return value of this function used?

Reply:
Not directly, If you remember in the UnitySDK implementation in GetAppPort() we set the LatestAppPort which is used in GetUrl()

Reply:
We should probably use the result of this function then. If there is an error, then wouldn't LatestAppPort not be set correctly?

Reply:
We throw an exception if AppPort was not found in GetAppPort(), I think the developer (me in this case) needs to catch the exception and act accordingly, Does that sound like the appropriate approach? 


<img width="511" alt="Screen Shot 2021-07-12 at 10 17 18 AM" src="https://user-images.githubusercontent.com/24863504/125329503-6d657880-e2fa-11eb-9652-7296a7ac432e.png">

Reply:
Gotcha. Ya that sounds good